scripts/xpp_update_happi_line

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+source pcds_conda
+cd $ENG_TOOL_SCRIPTS
```

Comment:
Where does this environment variable get set? I don't have it in my terminal.

Reply:
For some reason I have this sourced, finding exactly where from is tough as always

https://github.com/pcdshub/epics-config/blob/1403e2c62876d4c0b469e687816b569ef09da4fe/common_dirs.sh#L32

Reply:
Huh, interesting. Note then that the wrapper here will always run the latest version of the python script that's been updated in that specific folder, unless you set it again yourself.